socorro/app/socorro_app.py

```diff
@@ -206,6 +216,12 @@ def fix_exit_code(code):
 
             config_manager.log_config(mylogger)
 
+            # Add version to crash reports
+            version = (revision_data.get('version') or
```

Comment:
This is a copy of ``socorro.lib.revision_data.get_version`` that doesn't read the JSON from disk again. This may be a mis-optimization. I'm not sure if ``get_version`` should take an optional ``revision_data``, or I should just close my eyes and call ``get_version`` 🙈.

Reply:
I went with the optional ``revision_data``.